diff --git a/src/GUI-QT/DialogUtil.h b/src/GUI-QT/DialogUtil.h
--- a/src/GUI-QT/DialogUtil.h
+++ b/src/GUI-QT/DialogUtil.h
@@ -48,7 +48,6 @@
 #include <qwt_thermo.h> /* S-Meter */
 
 class CRig;
-typedef int rig_model_t;
 
 /* Definitions ****************************************************************/
 
diff --git a/src/util/Hamlib.cpp b/src/util/Hamlib.cpp
index 0fd823e..d544e77 100644
--- a/src/util/Hamlib.cpp
+++ b/src/util/Hamlib.cpp
@@ -424,7 +424,7 @@ CHamlib::LoadSettings(CSettings & s)
 		SetHamlibModelID(model);
 	}
 
-	s.Put("Hamlib", "hamlib-model", model);
+	s.Put("Hamlib", "hamlib-model", (int) model);
 	s.Put("Hamlib", "hamlib-config", strHamlibConf);
 	s.Put("Hamlib", "settings", strSettings);
 	s.Put("Hamlib", "freqoffset", iFreqOffset);
@@ -434,7 +434,7 @@ void
 CHamlib::SaveSettings(CSettings & s)
 {
 	/* Hamlib Model ID */
-	s.Put("Hamlib", "hamlib-model", iHamlibModelID);
+	s.Put("Hamlib", "hamlib-model", (int) iHamlibModelID);
 
 	/* Hamlib configuration string */
 	stringstream ss;
