diff --git a/dream.pro b/dream.pro
--- a/dream.pro
+++ b/dream.pro
@@ -3,8 +3,6 @@
 TARGET = dream
 OBJECTS_DIR = obj
 DEFINES += EXECUTABLE_NAME=$$TARGET
-LIBS += -L$$PWD/lib
-INCLUDEPATH += $$PWD/include
 contains(QT_VERSION, ^4\\..*) {
     VERSION_MESSAGE = Qt 4
 }
@@ -402,8 +400,8 @@
         LIBS += -lqwt-qt5
     }
     !crosscompile {
-        exists(/usr/include/qwt/qwt.h) {
-            INCLUDEPATH += /usr/include/qwt
+        exists(/usr/include/qt5/qwt/qwt.h) {
+            INCLUDEPATH += /usr/include/qt5/qwt
         }
         exists(/usr/local/include/qwt/qwt.h) {
             INCLUDEPATH += /usr/local/include/qwt
