diff --git a/dream.pro b/dream.pro
index 384bfda..974a902 100644
--- a/dream.pro
+++ b/dream.pro
@@ -3,8 +3,6 @@ CONFIG += warn_on
 TARGET = dream
 OBJECTS_DIR = obj
 DEFINES += EXECUTABLE_NAME=$$TARGET
-LIBS += -L$$OUT_PWD/lib
-INCLUDEPATH += $$OUT_PWD/include
 contains(QT_VERSION, ^4\\..*) {
     CONFIG += qt qt4
     VERSION_MESSAGE = Qt 4
@@ -144,7 +142,7 @@ unix:!cross_compile {
       packagesExist(gpsd) {
         CONFIG += gps
       }
-      packagesExist(pcap) {
+      exists(/usr/include/pcap.h) {
         CONFIG += pcap
       }
       packagesExist(opus) {
@@ -225,13 +223,13 @@ win32 {
          LIBS += -lz
     }
 }
-exists($$OUT_PWD/include/neaacdec.h) {
+exists(/usr/include/neaacdec.h) {
      DEFINES += HAVE_LIBFAAD \
      USE_FAAD2_LIBRARY
      LIBS += -lfaad_drm
      message("with FAAD2")
 }
-exists($$OUT_PWD/include/faac.h) {
+exists(/usr/include/faac.h) {
      DEFINES += HAVE_LIBFAAC \
      USE_FAAC_LIBRARY
      LIBS += -lfaac_drm
@@ -298,7 +296,7 @@ qwt {
             LIBS += -lqwtd
         } else {
             # unix | win release
-            LIBS += -lqwt
+            LIBS += -lqwt-qt5
         }
    }
    !crosscompile {
@@ -327,8 +325,8 @@ qwt {
             }
         }
     }
-    exists($$OUT_PWD/include/qwt/qwt.h) {
-        INCLUDEPATH += $$OUT_PWD/include/qwt
+    exists(/usr/include/qt5/qwt/qwt.h) {
+        INCLUDEPATH += /usr/include/qt5/qwt
     }
 }
 alsa {
